addons/mail_bot/models/res_users.py

```diff
@@ -25,12 +26,17 @@ class ResUsers(models.Model):
     def SELF_READABLE_FIELDS(self):
         return super().SELF_READABLE_FIELDS + ['odoobot_state']
 
-    def _init_messaging(self, store):
+    @api.model
+    def _on_webclient_bootstrap(self):
+        if self.odoobot_state in [False, 'not_initialized'] and self.env.user and self.env.user._is_internal():
+            self.env.user._init_odoobot()
+
+    @api.model
+    def _init_store_data(self, store: Store, /):
```

Comment:
And both methods should actually be made to work with `self` user rather than using the user from env internally.

Caller should just use `self.env.user._on_webclient_bootstrap()` as appropriate.